### ISSUE TITLE
cant install apk

### ISSUE THREAD

**Issue by bqMoonlight** [jspw/Ubuntu-Launcher#14]
It always says there was a problem with parsing the file,while i can install other apks with no issues.

I enables u known sources.
Samsung galaxy a52s 5g

**Comment by jspw** [comment]
which apk did you try ? @bqMoonlight

**Comment by jspw** [comment]
there are 3 apk files here. Different phones having different cpu architectures. Please try them all to know which one supports your phone.

**Comment by jspw** [comment]
most possibile [arm v8a](https://github.com/jspw/Ubuntu-Launcher/releases/download/2.4.0/app-arm64-v8a-release.apk)  will support in your phone.

**Comment by bqMoonlight** [comment]
I tryed all of them,its always the same issue. I changed permissions 2 times,and tryed new pipe apk to test,new pipe apk works,your gives me always the same error.

**Comment by jspw** [comment]
Ok I am checking this then..

**Comment by jspw** [comment]
Can you try this one ? [apk](https://drive.google.com/file/d/1PDYWPaqVSYqRp-d4CMpBMAadSTT5Izoh/view?usp=sharing)

**Comment by bqMoonlight** [comment]
Same issue...
Can i auto update from 2.3 to 2.4?

**Comment by jspw** [comment]
autually auto update is not possible for now as the app is not in a playstore... can you please check this again ? [apk](https://drive.google.com/file/d/1PDYWPaqVSYqRp-d4CMpBMAadSTT5Izoh/view?usp=sharing)

**Comment by bqMoonlight** [comment]
Same...

**Comment by bqMoonlight** [comment]
![Screenshot_20220310-120710_F-Droid](https://user-images.githubusercontent.com/97843623/157649844-4e72d839-37e3-4c58-885c-ba4e53e3ec6a.jpg)

This happens when i try using f droid

**Comment by AnaxMorden** [comment]
Same Error on Android S+

Tried with armv8a


![This is the detailed Error Message](https://user-images.githubusercontent.com/79524821/160291459-e6a29198-c270-42ac-a345-83f42bef8ef4.JPG)

**Comment by jspw** [comment]
Can you please try this [versions](https://drive.google.com/drive/folders/1cj3CPFrVJXNjKmJYvWm6Wr4bBlWQRY3P?usp=sharing) @AnaxMorden  @bqMoonlight

**Comment by AnaxMorden** [comment]
It seems that Theres now an New similar Problem

Error Log:
![IMG_20220328_082334](https://user-images.githubusercontent.com/79524821/160338434-b6e6cf04-c1b4-4e89-a3de-e38727f13c4d.jpg)

**Comment by SievaFliesAlone** [comment]
Same issue.

`W PackageInstaller: Parse error when parsing manifest. Discontinuing installation`

For all APKs (`v8a`, `v7a`, `x86`).
Pixel 6, Android 12.

**Comment by Anax-M** [comment]
@anibyl Could you please give us your API Level?
You can find it with the APP [AIDA64](https://play.google.com/store/apps/details?id=com.finalwire.aida64) in Section Android

And it needs to be tested on Android 11 so that we can verify its an Problem with the Android API

**Comment by SievaFliesAlone** [comment]
The API level is 32.

**Comment by Fenyx88** [comment]
Any update on this, on a galaxy z fold and am unable to install apk.

**Comment by Anax-M** [comment]
> Any update on this, on a galaxy z fold and am unable to install apk.

Nope nothing, seems a bit abandoned

**Comment by luckkmaxx** [comment]
Same here. Poco F3 (alioth) - EvolutionX ROM (A13)

**Comment by Anax-M** [comment]
Can someone try to install it on A9 or lower?

**Comment by avelblood** [comment]
> Can someone try to install it on A9 or lower?

I can try it on old Note 3.

Currently can't install arm8a apk onto Android 12 Snap 888 with the same error message as stated above.

**Comment by Anax-M** [comment]
> > Can someone try to install it on A9 or lower?
> 
> I can try it on old Note 3.
> 
> Currently can't install arm8a apk onto Android 12 Snap 888 with the same error message as stated above.

Yes because it seems that this Programm was not Designer for devices above A9, it would be nice if someone could confirm this

**Comment by jspw** [comment]
Sorry for late reply and thanks for your patience. Actually I don't have any physical device to test the launcher in Android 11 or higher. I was busy in some personal stuffs and will continue to contribute regularly. If anyone have any solution or idea please feel free to share.

**Comment by Anax-M** [comment]
> Sorry for late reply and thanks for your patience. Actually I don't have any physical device to test the launcher in Android 11 or higher. I was busy in some personal stuffs and will continue to contribute regularly. If anyone have any solution or idea please feel free to share.

Try that here from Android:
https://developer.android.com/about/versions/11/migration

Vor maybe do Android 12/13 directly

**Comment by jspw** [comment]
Thanks @AnaxMorden , Will look into it.

**Comment by 900c3** [comment]
> Thanks @AnaxMorden , Will look into it.

Any updates

**Comment by Anax-M** [comment]
> > Thanks @AnaxMorden , Will look into it.
> 
> Any updates

Currently not, you will See updates in this issue

**Comment by iKiwed** [comment]
So... Any updates?
